### ISSUE TITLE
Fresh lunarvim is very broken

### ISSUE THREAD

**Issue by rlpowell** [LunarVim/LunarVim#3684]
### Problem description

This is 100% reproducible like so:

`docker run --rm -it docker.io/library/debian:bullseye`

In that docker run the following:

```
apt-get update && apt-get install -y less lld clang git make python3-pip npm nodejs locales locales-all wget curl
cd /tmp/ && wget https://github.com/neovim/neovim/releases/download/v0.8.2/nvim-linux64.deb && apt install -y ./nvim-linux64.deb
wget -qO - https://sh.rustup.rs | RUSTUP_HOME=/opt/rust CARGO_HOME=/opt/rust sh -s -- --no-modify-path -y
source "/opt/rust/env"
rustup default stable
LV_BRANCH='release-1.2/neovim-0.8' bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/master/utils/installer/install.sh) -y
/root/.local/bin/lvim
```

This gets you something very similar to my actual setup, and you're in lunarvim, hooray!  But.

`:LvimInfo` gives:

```Error executing Lua callback: vim/shared.lua:0: t: expected table, got string
stack traceback:
        [C]: in function 'error'
        vim/shared.lua: in function 'validate'
        vim/shared.lua: in function <vim/shared.lua:0>
        vim/shared.lua: in function 'tbl_filter'
        /root/.local/share/lunarvim/lvim/lua/lvim/core/info.lua:115: in function 'make_auto_lsp_info'
        /root/.local/share/lunarvim/lvim/lua/lvim/core/info.lua:159: in function 'toggle_popup'
        /root/.local/share/lunarvim/lvim/lua/lvim/core/commands.lua:29: in function </root/.local/share/lunarvim/lvim/lua/lvim/core/commands.lua:28>
Press ENTER or type command to continue
```

, which is not a great look.

`:LvimCacheReset` gives:

```
Error executing Lua callback: /root/.local/share/lunarvim/lvim/lua/lvim/lsp/templates.lua:70: bad argument #1 to 'ipairs' (table expected, got string)
stack traceback:
        [C]: in function 'ipairs'
        /root/.local/share/lunarvim/lvim/lua/lvim/lsp/templates.lua:70: in function 'generate_templates'
        /root/.local/share/lunarvim/lvim/lua/lvim/utils/hooks.lua:48: in function 'reset_cache'
        /root/.local/share/lunarvim/lvim/lua/lvim/core/commands.lua:35: in function </root/.local/share/lunarvim/lvim/lua/lvim/core/commands.lua:34>
Press ENTER or type command to continue
packer.nvim: Error running config for lir.nvim: /root/.local/share/lunarvim/lvim/lua/lvim/core/lir.lua:111: attempt to call field 'setup' (a nil value)
packer.nvim: Error running config for mason.nvim: /root/.local/share/lunarvim/lvim/lua/lvim/core/mason.lua:86: attempt to call field 'setup' (a nil value)
Press ENTER or type command to continue
packer.nvim: Error running config for indent-blankline.nvim: ....local/share/lunarvim/lvim/lua/lvim/core/indentlines.lua:35: attempt to call field 'setup' (a nil value)
Press ENTER or type command to continue
packer.nvim: Error running config for telescope.nvim: ...t/.local/share/lunarvim/lvim/lua/lvim/core/telescope.lua:147: module 'telescope.previewers' not found:
^Ino field package.preload['telescope.previewers']
^Ino file './telescope/previewers.lua'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/share/luajit-2.1.0-beta3/telescope/previewers.lua'
^Ino file '/usr/local/share/lua/5.1/telescope/previewers.lua'
^Ino file '/usr/local/share/lua/5.1/telescope/previewers/init.lua'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/share/lua/5.1/telescope/previewers.lua'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/share/lua/5.1/telescope/previewers/init.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/telescope/previewers.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/telescope/previewers/init.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/telescope/previewers.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/telescope/previewers/init.lua'
^Ino file './telescope/previewers.so'
^Ino file '/usr/local/lib/lua/5.1/telescope/previewers.so'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/lib/lua/5.1/telescope/previewers.so'
^Ino file '/usr/local/lib/lua/5.1/loadall.so'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/lib/lua/5.1/telescope/previewers.so'
^Ino file './telescope.so'
^Ino file '/usr/local/lib/lua/5.1/telescope.so'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/lib/lua/5.1/telescope.so'
^Ino file '/usr/local/lib/lua/5.1/loadall.so'
Press ENTER or type command to continue
```

, and , umm , a bunch of other stuff.  It  goes on for A While.

Hey, maybe it's a Packer problem!, so:

`:PackerSync` gives:

```^Ino field package.preload['which-key']
^Ino file './which-key.lua'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/share/luajit-2.1.0-beta3/which-key.lua'
^Ino file '/usr/local/share/lua/5.1/which-key.lua'
^Ino file '/usr/local/share/lua/5.1/which-key/init.lua'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/share/lua/5.1/which-key.lua'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/share/lua/5.1/which-key/init.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/which-key.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/share/lua/5.1/which-key/init.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/which-key.lua'
^Ino file '/root/.cache/lvim/packer_hererocks/2.1.0-beta3/lib/luarocks/rocks-5.1/which-key/init.lua'
^Ino file './which-key.so'
^Ino file '/usr/local/lib/lua/5.1/which-key.so'
^Ino file '/home/runner/work/neovim/neovim/.deps/usr/lib/lua/5.1/which-key.so'
^Ino file '/usr/local/lib/lua/5.1/loadall.so'
Press ENTER or type command to continue
```

, and a bunch more errors, but eventually does what appears to be a bunch of installs.

After that, and a restart, things *do* appear to work, but this is very emphatically not what I expect after a fresh install.

### LunarVim version

release-1.2/neovim-0.8-8f4a7bd

### Neovim version (>= 0.8.0)

NVIM v0.8.2 Build type: Release LuaJIT 2.1.0-beta3 Compiled by runner@fv-az190-151  Features: +acl +iconv +tui See ":help feature-compile"     system vimrc file: "$VIM/sysinit.vim"   fall-back for $VIM: "/share/nvim"  Run :checkhealth for more info

### Operating system/version

Debian Bullseye

### Steps to reproduce

See main problem description.

**Comment by LostNeophyte** [comment]
were there errors during the installation?

**Comment by LostNeophyte** [comment]
for me in the debian container the installation errored out on `xdg-desktop-menu: No writable system menu directory found` and the installation didn't finish

**Comment by LostNeophyte** [comment]
could you check if this installer fixes the issue? `LV_BRANCH='release-1.2/neovim-0.8' bash <(curl -s https://raw.githubusercontent.com/LostNeophyte/lunarvim/fix/installer/utils/installer/install.sh)`

**Comment by rlpowell** [comment]
AaaaaAAaaaah, I didn't realize that the xdg error, which I ignored since I do not run Linux with a GUI and I get irrelevant errors like that all the time, was actually *stopping the install*, that makes perfect sense!

And it totally worked, thank you!
